pkg/store/storepb/labeladapter.go

```diff
@@ -0,0 +1,203 @@
+package storepb
```

Comment:
Curious if it's not the time to move this outside of storepb (rules has labels as well)